README.md

```diff
@@ -70,14 +70,12 @@ using (var scope = logger.BeginScope(dictionary))
 The `BeginScope` method allow any object, but only some of types are handled in an special way. Those types are:
 
 * `string`
-* `IEnumerable<KeyValuePair<string, string>>`
 * `IEnumerable<KeyValuePair<string, object>>`
 
-By default, any other type will be managed as a conventional `object`.
 
 #### How it works
 
-When you use any of the `IEnumerable<KeyValuePair<,>>` allowed types, the collection will be processed and every item from this collection should be introduced to the `LogicalThreadContext` managed by the log4net library using the `Key` as the property name and the `Value` as the value to replace the placeholder on the Pattern Layout defined.
+When you use any of the `IEnumerable<KeyValuePair<,>>` allowed types, the collection will be processed and every item from this collection should be introduced to the [`ExternalScopeProvider`](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.loggerexternalscopeprovider) using the `Key` as the property name and the `Value` as the value to replace the placeholder on the Pattern Layout defined.
```

Comment:
I'm not the maintainer so you should wait for other feedback before changing anything, but:

I think this should focus on that the scopes are added to the LoggingEvent Properties and maybe that it could be modified in the LoggingEventFactory.